_documentation/saml-azure.md

```diff
@@ -96,25 +97,18 @@ title: Login with Azure
 entityId: 'https://sts.windows.net/****-****-***/'                # Azure AD Identifier
 singleSignOnService:
     url: 'https://login.microsoftonline.com/****-****-***/saml2'  # Login URL
-    binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect'
-singleLogoutService:
-    url: 'https://login.microsoftonline.com/****-****-***/saml2'  # Logout URL
-    binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect'
 ```
 
-3. Change the **sp** configuration:
+3. Change the **sp** configuration replacing **https://timetracking.example.com** with your Kimai URL:
 ```yaml
 entityId: 'https://timetracking.example.com/auth/saml/metadata'
 assertionConsumerService:
     url: 'https://timetracking.example.com/auth/saml/acs'
-    binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST'
-# logout does not seem to work properly
-#singleLogoutService:
-#    url: 'https://timetracking.example.com/auth/saml/logout'
-#    binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect'
+singleLogoutService:
```

Comment:
> on step two I suggest to remove singleLogoutService since its not present in the example config above and I also don't recommend using it because on kimai logout you normally don't want to get logged out of Microsoft 365 as well

Why did you remove the comment then?

Reply:
sorry should have been clearer  - I removed the singleLogoutService from the **IDP** part but I left it in the **SP** part because the comment said "seems to not work" but in fact it was not working for IDP but I see no errors with my config where singleLogoutService  is present for SP

Reply:
Ah right, now I see the change. Diffs are sometimes weird to read...

You removed the `binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect'` for the `singleSignOnService` - did you try that it still works?

Reply:
yes I removed the binding since it worked also without it..